docs/security-considerations.rst

```diff
@@ -246,12 +246,8 @@ implications, there might be another issue buried beneath it.
 Any compiler warning we issue can be silenced by slight changes to the
 code.
 
-Also try to enable the "0.5.0" safety features as early as possible
-by adding ``pragma experimental "v0.5.0";``. Note that in this case,
-the word ``experimental`` does not mean that the safety features are in any
-way risky, it is just a way to enable some features that are
-not yet part of the latest version of Solidity due to backwards
-compatibility.
+Always use the latest version of the compiler to be notified about all recently
```

Comment:
The grammar seems to be a bit off here. Perhaps change it to ``Always use the latest version of the compiler to be notified about all warnings introduced recently``.